src/index.d.ts

```diff
@@ -4,6 +4,7 @@ declare module 'react-lazily-render' {
 
   export interface LazilyRenderProps {
     className?: string;
+    tagName?: string;
```

Comment:
Sorry to bikeshed over the name again, could we please change it to `component` and the type to `string | React.ComponentClass` 🙏